### ISSUE TITLE
Using an older version of the pack as a base for a custom version.

### ISSUE THREAD

**Issue by TheQuantumRuby** [kreezxil/World-of-Dragons#446]
Hi. I was wondering if I could use a very old version of the pack (the final 1.16 version, the last version before things such as Wizardry and Cuisine were added) as the base of my own pack (to possibly distribute.) I would, of course, link back to the original pack and say the version & credit you. I would also change text relating to the creator to 'Original pack made by Kreezxil' to properly credit you. Finally, if permission is granted, I will make a comment to this post with a link to the pack. I wanted to make this issue to make sure I wouldn't upset anyone. Thank you!

**Comment by kreezxil** [comment]
Yes you may and while not a requirement, it will greatly benefit myself and
other CF content creators of whom their mods are in the pack, that if you
do this that your pack is also hosted on CurseForge and preferably nowhere
else.

If you don't know by now CF has been giving out curse points for its entire
existence to content creators, this really took off when they got the
ability to host minecraft content, so for the last 3 years. Those points
are cashed in monthly. For example the JEI and Iron Chest devs make nearly
$2k USD monthly (undisclosed source). However, it's the little guys that
seldom get into a pack that would be hurt if the pack was hosted somewhere
else.

More info at
https://authors.curseforge.com/knowledge-base/rewards/3448-how-to-join-and-redeem-rewards
